app/scripts/directives/overview/serviceInstanceRow.js

```diff
@@ -130,5 +134,10 @@
     row.deprovision = function() {
       ServiceInstancesService.deprovision(row.apiObject, row.deleteableBindings);
     };
+
+    row.canAddMobileClient = function() {
```

Comment:
I guess this isn't dynamic? i.e. It won't update when a new client is created?
We could move it to an `$onChanges` function